documentation.go

```diff
@@ -26,7 +26,7 @@ This command generates a markdown formatted document with all the commands, thei
 `
 
 var documentationExamples = `
-    juju documentation
```

Comment:
While we're here, we should fix the formatting of this examples section so it is valid Markdown.
- Code blocks should be indented by four spaces.
- Non-code paragraphs should not be indented at all.

Curious that the `juju documentation` command does not generate documentation for itself... we'll have to fix that at some point.

Reply:
Done